subworkflows/nf-scil/registration/meta.yml

```diff
@@ -0,0 +1,80 @@
+name: "registration"
+description: |
+  Subworkflow to perform registration between a moving and a fixed image (e.g. T1 -> DWI).
+  It requires as input at least a moving (ch_image) and a reference image (ch_ref) to properly
+  perform registration. Two modes are available:
+  1)  if a metric file is supplied (ch_metric), the subworkflow will use the REGISTER_ANATTODWI
+      module calling AntsRegistration and
+  2)  if a metric file is NOT supplied, the subworkflow will use the REGISTRATION_ANTS module calling
+      antsRegistrationSyN.sh or antsRegistrationSyNQuick.sh.
+  Typical next steps after this subworkflow would be ANATOMICAL_SEGMENTATION or TRACKING.
```

Comment:
Here, I was more talking about what to do with the transformations computed and how to use the `ANTSAPPLYTRANSFORMS` module on images or tractograms.